proto/nodeinstance/v1/connection.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+option go_package = "nodeinstance/v1;connection";
+
+package nodeinstance.v1;
+
+import "google/protobuf/timestamp.proto";
+
+message UpdateNodeInstanceConnection {
```

Comment:
[question] Would it make sense to also include the `hops` in the payload?
Or hops for a nodeInstance should be computed every time in real time? 🤔 

https://dbdiagram.io/d/5efcac1d0425da461f041f33

Reply:
If there is no point for the `hops` to be included in the payload ☝🏼 
I think we're good with this definition (and the `NodeInstanceConnectionUpdated` one) and that we could proceed merging it @nktsitas @lvrach